ballerina/tests/08-procedure-call-test.bal

```diff
@@ -256,6 +273,24 @@ isolated function testCallWithNumericTypesOutParams() returns error? {
     test:assertEquals(paraBinDouble.get(float), 21474836.47, "4th out parameter of procedure did not match.");
     boolean|sql:Error status = ret.getNextQueryResult();
     test:assertTrue(status is boolean, "state is not a boolean");
+    check oracledbClient.close();
+}
+
+type Xml xml;
+
+@test:Config {
+    groups: ["procedures"]
+}
+isolated function testCallWithComplexTypesOutParams() returns error? {
+    Client oracledbClient = check new (HOST, USER, PASSWORD, DATABASE, PORT);
+    sql:IntegerValue paraID = new (1);
+    XmlOutParameter paraXml = new ();
+
+    var ret = check oracledbClient->call(
+        `{call SelectComplexDataWithOutParams(${paraID}, ${paraXml})}`);
+    check oracledbClient.close();
+    xml 'xml = xml `<key>value</key>`;
+    test:assertEquals(check paraXml.get(Xml), 'xml , "1st out parameter of procedure did not match.");
```

Comment:
Can't we just pass Ballerina `xml` type to the get method like `paraXml.get(xml)` rather than defining `Xml` type

Reply:
According to the @MaryamZi we can not pass the `xml` as a expression. 

here we can use `xml xmlValue = check paraXml.get()` if we want since I have define the XmlOutParameter as below,
```ballerina
# Represents Xml range OutParameter used in procedure calls
public distinct class XmlOutParameter {
    *sql:OutParameter;

    # Parses returned SQL value to ballerina value.
    #
    # + typeDesc - Type description of the data that need to be converted
    # + return - The converted ballerina value or Error
    public isolated function get(typedesc<anydata> typeDesc = <>) returns typeDesc|sql:Error = @java:Method {
        'class: "io.ballerina.stdlib.oracledb.nativeimpl.OutParameterProcessor"
    } external;
}

```